scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -0,0 +1,208 @@
+    input {
+        String project_id
+        String dataset
+    }
+    command <<<
+        python3 <<FIN
+
+        from google.cloud import bigquery
+
+        client = bigquery.Client(project="~{project_id}")
+        sample_info_table_id = f'~{project_id}.~{dataset}.sample_info'
+        sample_info_table = client.get_table(sample_info_table_id)
+
+        print(str(sample_info_table.num_rows))
+
+        FIN
+    >>>
+
+    output {
+        Int num_samples = read_int(stdout())
+    }
+    runtime {
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:ah_var_store_2022_08_01"
+    }
+}
+
+
+task ExtractFromNonSuperpartitionedTables {
+    meta {
+        description: "Extracts from the non-superpartitioned tables: sample_info, filter_set_info, filter_set_sites"
+    }
+    input {
+        String project_id
+        String dataset
+        String filter_set_name
+        String avro_sibling
+    }
+    parameter_meta {
+        avro_sibling: "Cloud path to a file that will be the sibling to the 'avro' 'directory' under which output Avro files will be written."
+    }
+    command <<<
+        set -o errexit -o nounset -o xtrace -o pipefail
+        echo "project_id = ~{project_id}" > ~/.bigqueryrc
+
+        avro_prefix="$(dirname ~{avro_sibling})/avro"
+        echo $avro_prefix > "avro_prefix.out"
+
+        bq query --nouse_legacy_sql --project_id=~{project_id} "
+            EXPORT DATA OPTIONS(
+            uri='${avro_prefix}/sample_mapping/sample_mapping_*.avro', format='AVRO', compression='SNAPPY') AS
+            SELECT sample_id, sample_name, '40',
+            'gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list' as intervals_file
+            FROM \`~{project_id}.~{dataset}.sample_info\`
+            ORDER BY sample_id
+        "
+
+        bq query --nouse_legacy_sql --project_id=~{project_id} "
+            EXPORT DATA OPTIONS(
+            uri='${avro_prefix}/vqsr_filtering/vqsr_filtering_*.avro', format='AVRO', compression='SNAPPY') AS
+            SELECT location, type as model, ref, alt, vqslod, yng_status
+            FROM \`~{project_id}.~{dataset}.filter_set_info\`
+            WHERE filter_set_name = '~{filter_set_name}'
+            ORDER BY location
+        "
+
+        bq query --nouse_legacy_sql --project_id=~{project_id} "
+            EXPORT DATA OPTIONS(
+            uri='${avro_prefix}/site_filtering/site_filtering_*.avro', format='AVRO', compression='SNAPPY') AS
+            SELECT location, filters
+            FROM \`~{project_id}.~{dataset}.filter_set_sites\`
+            WHERE filter_set_name = '~{filter_set_name}'
+            ORDER BY location
+        "
+
+        bq query --nouse_legacy_sql --project_id=~{project_id} "
+            EXPORT DATA OPTIONS(
+            uri='${avro_prefix}/vqsr_tranche/vqsr_tranche_*.avro', format='AVRO', compression='SNAPPY') AS
+            SELECT model, truth_sensitivity, min_vqslod, filter_name
```

Comment:
I _think_ we're going to want to drop filter_name here, but I'm adding it to the questions for Tim this afternoon